tests/libc/enc/enc.c

```diff
@@ -111,7 +111,7 @@ int run_test(const char* name, int (*main)(int argc, const char* argv[]))
 
     /* Allocate an environment for invoking the test. */
     {
-        if (!(environ = (char**)calloc(1, sizeof(char**))))
+        if (!(environ = (char**)calloc(1, sizeof(char*))))
```

Comment:
why change this? They're functionally equivalent and its general practice to use the sizeof the type you're allocating

Reply:
This is not resolved

Reply:
I know char* and char** are both pointers, but the meanings are different, and it does generate 10 related sizeof operand mismatch. Therefore I am wondering why char** is designed to point to an array of one char** element.

Reply:
oh, sorry, you're right. Re-resolving :smile: